namespaces/cert-manager/networkpolicy.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              namespace: kube-system
+        - namespaceSelector:
+            matchLabels:
+              namespace: cert-manager
```

Comment:
I havent had a look in details how the namespaces are created, but make sure that the namespace has the label `namespace: cert-manager` which is not there by default. same applies to all other labels